trip_app/trips/tests/test_models.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from trips.models import TripRequest
+from trips.tests.trip_factory import TripRequestFactory
+
+
+@pytest.mark.django_db
```

Comment:
These test cases could be within a class, with `@django_db` used for all of them once.

Reply:
Should be updated now.